src/chip-input/ux-chip-input.html

```diff
@@ -0,0 +1,22 @@
+<template role="textbox" styles.chipinput>
+  <require from="./ux-chip-input-theme"></require>
+
+    <ux-chip deletable
+             ref="chiprepeat"
+             close.trigger="removeChip(chip)"
+             dblclick.trigger="editChip(chip)"
+             if.bind="type.toLowerCase() !== 'tag'"
+             repeat.for="chip of chipArray">
+             ${chip}
+             </ux-chip>
+    <ux-tag deletable
+            ref="tagrepeat"
+            close.trigger="removeChip(chip)"
+            dblclick.trigger="editChip(chip)"
+            if.bind="type.toLowerCase() === 'tag'"
```

Comment:
Ah, do you know what happens, just for my own curiosity? I went ahead and changed this.